tinymce6-parent/tinymce6-examples/LICENSE.txt

```diff
@@ -0,0 +1,165 @@
+		   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
copy paste from tinymce4

Reply:
I am not the best person to tell how it should be but I think the license of this package should be ASL-2 and a NOTICE file should say that it includes tinymce6 (the JS package) that is licensed with XYZ.
Even better it would be if the JS package is included as a webjar, i.e. a Maven dependency.

Reply:
I found a webjar and replaced the Checked in Tinymce sources completely.